### DISCUSSION TITLE
TTF fonts in codesandbox

### DISCUSSION THREAD

**Discussion by Joshua131313** [codesandbox/codesandbox-client#5900] [Q&A]
How can I use a ttf font in my React App on codesandbox?

**Comment by alexnm** [comment]
Hey! If you have ttf or any other format of font files you can upload them in codesandbox as static files. You can drag and drop a file in the file explorer or use the file upload button at the top right corner 
![image](https://user-images.githubusercontent.com/9945366/122174007-07b4d680-ce8b-11eb-9eff-3a0632a244b6.png)

If you are using create-react-app, the `public` folder should keep all static files for the runtime.

**Reply by Joshua131313** [reply]
Hey, thanks for the reply, I actually found out that when using the domain provided by codesandbox, the font is not loaded, but when i push it with vercel, the fonts get loaded.

**Reply by alexnm** [reply]
can you share an example?
